ngraph_bridge/CMakeLists.txt

```diff
@@ -86,7 +86,12 @@ endif()
 
 
 if(NGRAPH_TF_USE_GRAPPLER_OPTIMIZER)            
-    list(REMOVE_ITEM SRC ngraph_rewrite_pass.cc)
+    if(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
+        list(REMOVE_ITEM SRC enable_variable_ops/ngraph_rewrite_pass.cc)
+    else()
+        list(REMOVE_ITEM SRC ngraph_rewrite_pass.cc)
+    endif()
+    list(APPEND SRC enable_variable_ops/ngraph_catalog.cc)
```

Comment:
Shouldn't this be only appended `if(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)` ?

Reply:
No, since ngraph_catalog is also needed for the grappler path.